### PR TITLE
Update the Regulations Translations page.

### PR DESCRIPTION
**Pull request by AlbertoPdRF**
Follow up of #4497.

image

### CODE REVIEW

File: WcaOnRails/app/views/regulations/translations/index.html.erb [hunk 1]
```diff
@@ -1,88 +1,143 @@
...
+          language_english: "Chinese",
+          url: "./chinese",
+        },
+        {
+          version: "2019-05-01",
+          language: "日本語",
+          language_english: "Japanese",
+          url: "./japanese",
+        },
+      ] %>
```
Comment (jonatanklosko) [inline WcaOnRails/app/views/regulations/translations/index.html.erb]: The indentation seems a bit of, what about:
```erb
<% current_translations = [
  {
    version: "2019-05-01",
    language: "简体中文",
    language_english: "Chinese",
    url: "./chinese",
  },
  {
    version: "2019-05-01",
    language: "日本語",
    language_english: "Japanese",
    url: "./japanese",
  },
] %>
```

File: .gitignore [hunk 2]
```diff
@@ -3,7 +3,7 @@ WcaOnRails/app/views/regulations/*.*
 WcaOnRails/app/views/regulations/translations/*
 WcaOnRails/app/views/regulations/scrambles/*
 WcaOnRails/app/views/regulations/history/*
-regulations/
+/regulations/
```
Comment (jfly) [inline .gitignore]: Just curious, is there a reason you had to change this? I thought they would behave the same.

Comment (jonatanklosko) [reply .gitignore]: Actually leading `/` indicates project root directory. Without that, it matches directory of the given name anywhere in the tree. In this case we want to ignore the top level regulations directory, but end up ignoring `WcaOnRails/app/views/regulations` as well.
We do have gitignore exceptions below, but if you ignore an entire directory then git don't really care about those exceptions, so we would need to `git add -f` every new file in that directory.
[Reference](https://git-scm.com/docs/gitignore/2.3.0):
> An optional prefix "!" which negates the pattern; any matching file excluded by a previous pattern will become included again. It is not possible to re-include a file if a parent directory of that file is excluded.

**Comment by lgarron** [comment]
I think this is worth shipping!

File: WcaOnRails/app/views/regulations/translations/index.html.erb [hunk 3]
```diff
@@ -1,88 +1,143 @@
...
+          version: "2014-01-01",
+          language: "Беларуская мова",
+          language_english: "Belarusian",
+          url: "./belarusian",
+        },
+      ] %>
+
+  <h3><%= t("regulations_translations.old") %></h3>
+  <div class="regulations-translations-table-container">
+    <%= render 'regulations/translations/index_table', translations: old_translations %>
```
[the remark this answers is not in the document]
Comment (AlbertoPdRF) [reply WcaOnRails/app/views/regulations/translations/index.html.erb]: I can't seem to get this working :confused:

File: WcaOnRails/app/views/regulations/translations/index.html.erb (hunk 1 above)
Comment (AlbertoPdRF) [reply WcaOnRails/app/views/regulations/translations/index.html.erb]: Ups, right!

**Comment by lgarron** [comment]
Thanks for landing this! :-D

**Comment by kotarot** [comment]
Hi,

Don't you keep the translators' names in the new page? Do you plan to add them later?
I think we should keep the translators' credit on the website respecting them, at least for the latest versions.

**Comment by lgarron** [comment]
> Hi,
> 
> Don't you keep the translators' names in the new page? Do you plan to add them later?
> I think we should keep the translators' credit on the website respecting them, at least for the latest versions.

I think that there are too many translators for the page at this point, and I think we should streamline the page to make it easy to find translations (no one goes there to find the names of translators).

I think it would be good to make sure each translation credits its translators at the top, though.

**Comment by kotarot** [comment]
@lgarron 

> I think we should streamline the page to make it easy to find translations (no one goes there to find the names of translators).

Yes, that's right... I agree with this part.

> I think it would be good to make sure each translation credits its translators at the top, though.

I like your idea. Do you think we firstly need to introduce something like `<translators>` tag like `<version>`?
https://github.com/thewca/wca-regulations-translations/blame/master/japanese/wca-regulations.md#L3

Or just adding a section for credits in the markdown is enough?

**Pull request merged**